.github/workflows/ci.yml

```
@@ -38,8 +38,9 @@ jobs:
   # Build Packages
 
   esp-hal:
-    name: esp-hal (${{ matrix.device.soc }})
+    name: esp-hal (${{ matrix.device.soc }} | ${{ matrix.device.toolchain }})
     runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.device.continue_on_error }}
```

```suggestion
    continue-on-error: ${{ matrix.device.toolchain == "nightly" }}
```

I think we can probably simplify this and not need to add all the additional metadata to the matrix?